src/pages/getStarted/components/StepSecond/index.tsx

```diff
@@ -154,6 +168,23 @@ const StepSecond = (props: IProps) => {
             })
           );
           break;
+        case 'Node.js':
+          saveLanguage('node');
```

Comment:
`node` is not an available option of react-syntax-highlighter
Use `javascript` instead